src/cortex_cli/auth.py

```diff
@@ -80,8 +80,10 @@ def login_request(url: str, realm: str, client_id: str, username: str, password:
 
     request_url = f'{url}/realms/{realm}/protocol/openid-connect/token'
     result = requests.post(request_url, data=data.dict(exclude_none=True), timeout=AUTH_REQUESTS_TIMEOUT)
+    if result.status_code == 404:
+        raise ClientAuthenticationError(f'token endpoint is not available at {url}')
     if result.status_code != 200:
-        raise ClientAuthenticationError(f'Failed to authenticate, {result.text}')
+        raise ClientAuthenticationError('invalid username and/or password')
```

Comment:
```suggestion
        raise ClientAuthenticationError('Invalid username and/or password')
```

Reply:
ClientAuthenticationError message from `login_request` is appended (as a further explanation) to "Failed to authenticate" message, so the messages here are all lowercase.

Reply:
Yeah, I realized that and deleted this comment a while ago, not sure why it's still here :-|